Added introduction and users topics (#50)
osisoft/PI-Adapter: V1/introduction-to-pi-adapters.md

### Inbound data

PI Adapters provide connections to a variety of inbound data sources from common industrial protocols. Currently available protocols include OPC UA, Modbus TCP, DNP3, and BACnet. Protocols in progress include MQTT, Azure Event Hubs, RDBMS, and Structured Data Files.

The optional stream metadata capability provides the option to set the amount of metadata to provide meaningful metadata that differs for every adapter type.

For more information, see [Data source configuration](xref:DataSourceConfiguration) and [Data selection configuration](xref: DataSelectionConfiguration).

### Data transfer

PI Adapters allow you to transfer real-time data to PI Server, OSIsoft Cloud Services (OCS), and Edge Data Store (EDS). The data filtering capability improves the configuration flexibility by manipulating data on the outbound side and minimizes network bandwidth consumption through exception reporting capabilities.

For more information, see [Egress configuration](xref:EgressConfiguration).

### Buffering

> PI Adapters incorporate robust data collection through buffering for data egressed from the adapter to egress endpoints or health endpoints. Buffering helps to overcome network and system disruptions that often result in data loss.

Author: @osisoft-mbishop Hmm, your suggestion changes the meaning of the sentence

> PI Adapters incorporate robust data collection through buffering for data egressed from the adapter to egress endpoints or health endpoints. Buffering helps to overcome network and system disruptions that often result in data loss.

Reviewer: @TinaUll , I was just floating a suggestion because I didn't understand the first sentence. "incorporate robust data collection" sounds generic. Feel free to decline.

- Edge Data Store (EDS) for additional edge applications 
- PI Server for on-premises storage 
- OSIsoft Cloud Services (OCS) for cloud-hosted data management service

PI Adapters offer the following generic capabilities:

- Connection to [inbound data](#inbound-data) sources from common industrial protocols
- Real-time [data transfer](#data-transfer) to PI Server, OCS, and EDS with optional data filtering to manipulate data on the outbound side
- [Buffering](#buffering) to overcome network and system disruptions
- [Administration](#administration) functions for simplified management, scalable installation, and configuration

## Capabilities in detail

The following capabilities are available for every PI Adapter. A specific PI Adapter might or might not include additional capabilities such as network proxy support or data source auto discovery.

### Inbound data

PI Adapters provide connections to a variety of inbound data sources from common industrial protocols. 

> The optional stream metadata capability provides the option to set the amount of metadata to provide meaningful metadata that differs for every adapter type.

Author: @osisoft-mbishop How about this?
```suggestion
The optional stream metadata capability allows to set the amount of metadata sent with each stream. Metadata are different for every adapter type.
```

Reviewer: I like that! :thumbsup:

It's missing a "you", but otherwise that's much more elegant than my suggestion:

>The optional stream metadata capability allows **you** to set the amount of metadata sent with each stream. Metadata are different for every adapter type.

Author: Great, thanks!
